test/unit/TestIOSXRDriver.py

```diff
@@ -57,7 +59,9 @@ class FakeIOSXRDevice:
 
     @staticmethod
     def read_txt_file(filename):
-        with open(filename) as data_file:
+        curr_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
```

Comment:
Why not just:

curr_dir = os.path.dirname(os.path.abspath(inspect.getfile(__file__)))